pkgs/by-name/ob/objdiff/package.nix

```diff
@@ -0,0 +1,89 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  stdenv,
+  pkg-config,
+  autoPatchelfHook,
+  darwin,
+  fontconfig,
+  wayland,
+  libxkbcommon,
+  libGL,
+  libX11,
+  libXcursor,
+  libXi,
+  zenity,
+}:
+let
+  version = "2.0.0";
+  commit = "3846a7d315a9d1e3ffaf3f917345ffb225b1b400";
```

Comment:
I would recommend to just mock this with an obvious fake value to make updates easier and less a chore.

Reply:
I'd rather keep it as-is, to minimize differences with upstream. I'll take care of changing it as things get updated, it's not a huge hassle.